src/test/scala/org/apache/mesos/chronos/scheduler/mesos/ConstraintCheckerSpec.scala

```diff
@@ -0,0 +1,85 @@
+package org.apache.mesos.chronos.scheduler.mesos
+
+import mesosphere.mesos.protos._
+import org.apache.mesos.chronos.scheduler.jobs.constraints.ConstraintSpecHelper
+import org.apache.mesos.Protos
+import java.util.logging.Logger
+import org.specs2.mock.Mockito
+import org.specs2.mutable.SpecificationWithJUnit
+import org.apache.mesos.chronos.scheduler.jobs.constraints.LikeConstraint
+import org.apache.mesos.chronos.scheduler.jobs.constraints.EqualsConstraint
+import mesosphere.mesos.protos.Implicits._
+import org.specs2.specification.BeforeEach
+
+/**
+ * spec for testing ConstraintChecker
+ *
+ * @author tony kerz (anthony.kerz@gmail.com)
+ */
+class ConstraintCheckerSpec extends SpecificationWithJUnit 
+with Mockito 
+with ConstraintSpecHelper 
+{
+  isolated
```

Comment:
it's been a little while, but i think i was under the impression that using the same `offer` across multiple tests was causing an issue so i wanted a new one created for each test and i was following advice given [here](https://gitter.im/etorreborre/specs2?at=5608b7cfefd97f3e4669bda7) to accomplish that.